services/core-api/app/api/mines/reports/models/mine_report_definition.py

```diff
@@ -241,12 +240,10 @@ def apply_filters_and_pagination(cls, query, page, per_page, sort_field, sort_di
         compliance_filter = True if regulatory_authority or section else False
 
         if compliance_sort or compliance_filter or not show_expired:
-            query = query.join(MineReportDefinitionComplianceArticleXref,
-                               MineReportDefinitionComplianceArticleXref.mine_report_definition_id == MineReportDefinition.mine_report_definition_id,
-                               isouter=True)
-            query = query.join(ComplianceArticle,
-                               ComplianceArticle.compliance_article_id == MineReportDefinitionComplianceArticleXref.compliance_article_id,
-                               isouter=True)
+            query = query.outerjoin(MineReportDefinitionComplianceArticleXref,
+                               MineReportDefinitionComplianceArticleXref.mine_report_definition_id == MineReportDefinition.mine_report_definition_id)
+            query = query.outerjoin(ComplianceArticle,
```

Comment:
Was having trouble figuring out exactly why, but doing the join with `isouter` was filtering out reports with no compliance code, but changing it to a `query.outerjoin` works properly.